clusters/hive/pools/cvp/install-config-aws-us-east-1_secret.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: install-config-aws-us-east-1
+  namespace: cvp-cluster-pool
+stringData:
+  # The public key is taken from https://github.com/openshift/release/blob/80d88fb8ab7eec427d001ed7fb67741251c49dcf/core-services/ci-secret-bootstrap/_config.yaml#L869-L871
```

Comment:
You should absolutely use a secret that you own, not one that we own

Reply:
I think you'll want this secret: https://github.com/openshift/release/blob/master/core-services/ci-secret-bootstrap/_config.yaml#L781

Reply:
OK, I will update stringData, but I must get access to that public key, so it can be specified at row 40.

Reply:
I'd maybe suggest to decouple from the secret in the cluster profile and just create (and maintain in Vault) a set of secrets solely to support your pools.